### ISSUE TITLE
Create new label for ResampleBenchmark

### ISSUE THREAD

**Issue by phcerdan** [InsightSoftwareConsortium/ITKPerformanceBenchmarking#59]
The new benchmark contributed by @dpshamonin is really great, but takes long time, I think it deserves its own label, instead of integrating it with `Filtering`, these are the result with everything enabled:

```
Label Time Summary:
Core                       =   0.03 sec*proc (1 test)
Filtering                  = 285.63 sec*proc (65 tests)
PerformanceBenchmarking    =   0.19 sec*proc (6 tests)
Registration               =   8.38 sec*proc (3 tests)
Segmentation               =  27.36 sec*proc (4 tests)
```

**Comment by thewtex** [comment]
:+1: good idea. A new CTest label and disabled by default so testing time is reasonable by default. This is important for exploring performance changes.

**Comment by blowekamp** [comment]
We should select at least 2-4 common or interesting cases which should still be enabled by default.

**Comment by dzenanz** [comment]
Yes, we should have the few quickest running tests on by default.

**Comment by blowekamp** [comment]
> Yes, we should have the few quickest running tests on by default.

Hmm, perhaps there are two things. There are the **Tests** which are run by the CI by default and then there is the **performance profile** done by the diver. Perhaps the different purpose dictates difference defaults.
